### PR TITLE
Fix missing "regenerator" support in workers

### PR DESCRIPTION
**Pull request by ibgreen**
This issue has been preventing the update of deck.gl examples to use loaders.gl.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
   "workspaces": [
     "modules/*"
   ],
+  "browser": {
+    "module-alias": false
```
Comment (Pessimistress) [inline package.json]: Why is this needed? Isn't module-alias a dev dependency?

File: babel.config.js [hunk 2]
```diff
@@ -10,38 +10,45 @@ const TARGETS = {
...
 CONFIG.es6 = Object.assign({}, CONFIG.default, {
   presets: [
     ['@babel/env', {
       targets: TARGETS,
       modules: false
     }]
-  ]
+  ],
+  plugins: PLUGINS.concat([
+    ['@babel/plugin-transform-runtime', {useESModules: true}]
```
Comment (Pessimistress) [inline babel.config.js]: My understanding is that babel runtime is used to replace polyfill. The `es6` targets surely do not need it?

File: package.json (hunk 1 above)
Comment (ibgreen) [reply package.json]: Yes it is. 

aliases is however now getting included in the browser-test build (to set the aliases for `readFile` which generates an ugly red error in the console log. I was working to remove all such noise from the log and this was one of the issues. 

Completely open to doing it in a different way... The call to moduleAlias should probably be moved out of `aliases.js` but was afraid of breaking something there.

File: babel.config.js (hunk 2 above)
Comment (ibgreen) [reply babel.config.js]: > My understanding is that babel runtime is used to replace polyfill. The `es6` targets surely do not need it?

The ES6 targets use babel-preset-env with a selection of browser and node versions. That removes all babel transforms that are supported by those versions, but still includes a few transforms to support e.g. Edge, Node 8, Safari. It is possible to set a verbose flag on preset-env to make it print the selected list.

In particular, now that we start using ES2018 syntax we will hit more of these (preset-env handles all transforms that have reached stage4, so it is presumably ES2018+growing subset of ES2019).

File: package.json (hunk 1 above)
Comment (ibgreen) [reply package.json]: Move `addAlias` to `test/start.js` and dropped the `package.json` config.
